src/js/components/atoms/li-item/template.vue.html

```diff
@@ -1,4 +1,5 @@
 <li class="list__item">
-    <template v-if="!item.item">{{ item }}</template>
-    <template v-else><component :is="item.type" v-bind="item.item"/></template>
+    <template v-if="item.item"><component :is="item.type" v-bind="item.item"/></template>
```

Comment:
This does not effect the first initial server side render? For the crawler magic to happen.

Reply:
resolved in discussion with @zwaardje